lib/galaxy/model/__init__.py

```diff
@@ -551,6 +551,7 @@ class WorkerProcess(Base, UsesCreateAndUpdateTime):
     hostname: Mapped[Optional[str]] = mapped_column(String(255))
     pid: Mapped[Optional[int]]
     update_time: Mapped[Optional[datetime]] = mapped_column(default=now, onupdate=now)
+    is_webapp: Mapped[Optional[bool]]
```

Comment:
Can you make this a string please so we can track workflow handlers, job handlers, celery tasks etc in the future ?